src/packing/pack_impl.c

```diff
@@ -164,7 +164,7 @@ __wt_struct_repack(WT_SESSION_IMPL *session, const char *infmt,
 	 * The former case has the size embedded before the item, the latter
 	 * does not.
 	 */
-	if ((len = strlen(outfmt)) > 1 && outfmt[len - 1] == 'u' &&
+	if ((len = strlen(outfmt)) > 0 && outfmt[len - 1] == 'u' &&
```

Comment:
@ddanderson, I know you're still bug hunting, but this part of the change confuses me.  Won't it turn ordinary "u" into "U"?  In other words, the common case of a `WT_ITEM` will need to be encoded with a size prefix.